docs/book/requirements.rst

```diff
@@ -0,0 +1,78 @@
+.. index::
+    single: System Requirements
+
+System Requirements
+===================
+
+Here you will find the list of system requirements that have to be adhered to be able to use **Sylius**.
+First of all have a look at the `requirements for running Symfony <http://symfony.com/doc/current/reference/requirements.html>`_.
+
+Read about the `LAMP stack <https://en.wikipedia.org/wiki/LAMP_(software_bundle)>`_.
+
+Operating Systems
+~~~~~~~~~~~~~~~~~
+
+The recommended operating systems for running Sylius are the Unix systems - **Linux, MacOS**.
+
+Web server and configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We do recommend using Apache web server ≥ 2.2.
+Remember to load the ``php`` module in the ``/etc/apache2/httpd.conf``.
+
+PHP required modules and configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Required**:
+
++---------------+-----------------------+
+| PHP           | ^5.6                  |
```

Comment:
It's not so clear, that we require PHP ^5.6 and recommend ^7.0, it's a version conflict :)